lib/node/fullnode.js

```diff
@@ -93,6 +93,7 @@ class FullNode extends Node {
       identityKey: this.identityKey,
       maxOutbound: this.config.uint('max-outbound'),
       maxInbound: this.config.uint('max-inbound'),
+      p2prps: this.config.uint('p2prps'),
```

Comment:
What do you think of making the config option be `p2p-rps`? I think that is a lot easier to read.